### ISSUE TITLE
Using XGBoost to make regression prediction but using 'argmax' in classifier instead.

### ISSUE THREAD

**Issue by BlackPoint-CX** [sql-machine-learning/sqlflow#795]
**Description**
Using XGBoost to make regression prediction but using 'argmax' in classifier instead.

**Reproduction Steps**
```
%%sqlflow
SELECT *
FROM datamap_test.ds_xgboost_test
PREDICT ds_xgboost_predict.duration
USING datamap_test.ds_xgboost_model ;
```

**Expected Behavior**
Make right regression.

**Screenshots**
```
[15:47:58] 8913x57 matrix with 508041 entries loaded from predict.txt

Traceback (most recent call last):

  File "<stdin>", line 618, in <module>

  File "/home/luban/miniconda/envs/sqlflow-dev-py36/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 1103, in argmax

    return _wrapfunc(a, 'argmax', axis=axis, out=out)

  File "/home/luban/miniconda/envs/sqlflow-dev-py36/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 56, in _wrapfunc

    return getattr(obj, method)(*args, **kwds)

numpy.AxisError: axis 1 is out of bounds for array of dimension 1

StatusCode.UNKNOWN
exit status 1
```

**Comment by Yancey0623** [comment]
Hi @BlackPoint-CX , xgboost prediction hasn't supported the regression model, will do that today.

**Comment by Yancey0623** [comment]
Hi @BlackPoint-CX , this issue has been fixed by #797 , please reference SQL in the unit test to verify it:  https://github.com/sql-machine-learning/sqlflow/blob/develop/cmd/sqlflowserver/main_test.go#L1006
